sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -73,6 +83,23 @@ public partial class ClientPipelineOptions
         public System.ClientModel.Primitives.PipelineTransport? Transport { get { throw null; } set { } }
         public void AddPolicy(System.ClientModel.Primitives.PipelinePolicy policy, System.ClientModel.Primitives.PipelinePosition position) { }
     }
+    public partial class ClientRetryPolicy : System.ClientModel.Primitives.PipelinePolicy
+    {
+        public ClientRetryPolicy() { }
+        public ClientRetryPolicy(int maxRetries) { }
+        protected virtual System.TimeSpan GetNextDelayCore(System.ClientModel.Primitives.PipelineMessage message, int tryCount) { throw null; }
```

Comment:
Are we planning to expose a public `GetNextDelay`? Otherwise, why do we need `Core` suffix?

Reply:
I think I have GetNextDelay as an internal method, and wanted to preserve the ability to add functionality that end-users couldn't override if needed by using the Template method.  I can revisit the Template method docs to see if it makes sense to expose Core if you don't expose the outer method publicly.